Assets/Scripts/EnemyPatrol.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyPatrol : MonoBehaviour
+{
+
+    [SerializeField]
+    private float moveSpeed, checkRadius, attackDelay, deathDelay, knockbackPower, knockbackHeight, knockbackTime;
+
+    public short enemyHealth;
+
+    private bool hittingWall, notEdge, playerInRange, moveRight, 
+        isAttacking, hasAttacked, attackedOnRight, canMove, isDamagable, beingKnockedBack;
+    
+    PlayerController playerController;
+
+    EnemyHealth EnemyHealthController;
+
+    CircleCollider2D enemyAttackPoint;
+
+    [SerializeField]
```

Comment:
Group all the serialized fields at the top, wrap them in a region labeled "serialized fields" https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region  

https://docs.google.com/document/d/1yrKCY_sVAdWkwgWAjJLiJFj1OOHn6dC__pGkSVB61YY/edit#heading=h.oezlj2pye3ju